### ISSUE TITLE
What is the shap_value output of dependence_plot ?

### ISSUE THREAD

**Issue by ManuelZ** [shap/shap#1367]
Thank you for creating SHAP!

I have a Catboost model with a loss function specified as LogLoss and a `dependence_plot` that talks a bout a SHAP value, but I'm not sure if it's referring to the log of the odds.

```
cv_params = {
    'eval_metric': 'AUC',
    'loss_function': 'Logloss',
    'custom_loss': ['AUC', 'F1'],
}
model = CatBoostClassifier(**cv_params)
model.fit(train_pool, eval_set=validation_pool)

explainer = shap.TreeExplainer(model, model_output='raw')
shap_values = explainer.shap_values(np.array(X_train))

shap.dependence_plot('start_year', shap_values, X_train, interaction_index='auto') 
```

- Is it referring to the log of the odds?
- In other functions like force_plot I can specify the `link='logit'` to obtain probabilities. Isn't that possible with `dependence_plot`?
